src/components/courses/CourseDetails.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="bg-white">
+        <div class="mx-auto max-w-2xl py-3 px-4 sm:py-5 sm:px-6 lg:grid lg:max-w-7xl lg:grid-cols-2 lg:gap-x-8 lg:px-8">
+            <!-- Product details -->
+            <div class="lg:max-w-lg lg:self-end">
+                <div class="mt-4">
+                    <h1 class="text-3xl font-bold tracking-tight text-gray-900 sm:text-4xl" id="title">{{ courseStore.course.title }}</h1>
+                </div>
+
+                <section aria-labelledby="information-heading" class="mt-4">
+                    <h2 id="information-heading" class="sr-only">Course information</h2>
+
+                    <div class="flex items-center">
+                        <p class="text-lg text-gray-900 sm:text-xl" id="duration">{{ courseStore.course.duration }} minutes</p>
+
+                        <div class="ml-4 border-l border-gray-300 pl-4">
+                            <h2 class="sr-only">Reviews</h2>
+                            <div class="flex items-center">
+                                <div>
+                                    <div class="flex items-center">
+                                        <StarIcon v-for="rating in [0, 1, 2, 3, 4]" :key="rating" :class="[courseStore.course.rating > rating ? 'text-yellow-400 course-star' : 'text-gray-300', 'h-5 w-5 flex-shrink-0']" aria-hidden="true" />
+                                    </div>
+                                    <p class="sr-only">{{ courseStore.course.rating }} out of 5 stars</p>
+                                </div>
```

Comment:
FYI: here's possible to remome extra div

Reply:
This div is made to place five stars in the centre